sktime/forecasting/compose/_pipeline.py

```diff
@@ -536,7 +535,7 @@ def _transform(self, X=None, y=None):
 #     return Zt
 
 
-class TransformedTargetForecaster(_Pipeline, _SeriesToSeriesTransformer):
+class TransformedTargetForecaster(_Pipeline):
```

Comment:
dont we need to deprecate the transformer interface and point to the `TransformerPipeline`?

Reply:
summary of our discussion on slack: the `transform` method is still there and functioning as before, so no deprecation is necessary. We may want to deprecate this in the future, and indeed replace by `TransformerPipeline`.